themes/beaver/layouts/partials/blog/footer.html

```diff
@@ -0,0 +1,58 @@
+</div>
+
+<footer class="footer">
+    <div class="footer-container">
+        <div class="flex-container">
+        <div class="info flex2">
+            <a href="/" title="Go to homepage"><img loading="lazy" width="200" height="36.5" alt="JetThoughts" src="/wp-content/uploads/2022/09/jet-thoughts-white.png"></a>
+            <p>{{ .Site.Params.description }}</p>
+            <p><a href="mailto:{{ .Site.Params.email }}">{{ .Site.Params.email }}</a></p>
+            <p><a href="tel:{{ .Site.Params.phone }}">{{ .Site.Params.phone }}</a></p>
+        </div>
+        <div class="services flex1">
+            <h3>Services</h3>
+            <ul>
+                {{ range .Site.Menus.services }}
+                <li><a href="/clients/">{{ .Name }}</a></li>
+                {{ end }}
+            </ul>
+        </div>
+        <div class="company flex1">
+            <h3>Company</h3>
+            <ul>
+                {{ range .Site.Menus.company }}
+                <li><a href="/clients/">{{ .Name }}</a></li>
+                {{ end }}
+            </ul>
+        </div>
+        <div class="resources flex1">
+            <h3>Resources</h3>
+            <ul>
+                {{ range .Site.Menus.resources }}
+                <li><a href="/clients/">{{ .Name }}</a></li>
+                {{ end }}
+            </ul>
+        </div>
+    </div>
+</div>
+    <div class="footer-bottom">
+        <p>{{ .Site.Params.copyright }}</p>
+        <p class="social-icons">
+            <a href="https://www.facebook.com/jetthoughts/" target="_blank" title="Facebook" aria-label="Facebook" role="button" rel="noopener noreferrer external">
+                {{ partial "svg" "mini/facebook" }}
+            </a>
+            <a href="https://twitter.com/jetthoughts/" target="_blank" title="Twitter" aria-label="Twitter" role="button" rel="noopener noreferrer external">
+                {{ partial "svg" "mini/x" }}
+            </a>
+            <a href="https://www.linkedin.com/company/jetthoughts/" target="_blank" title="LinkedIn" aria-label="LinkedIn" role="button" rel="noopener noreferrer external">
+                {{ partial "svg" "mini/linkedin" }}
+            </a>
+            <a href="https://github.com/jetthoughts/" target="_blank" title="Github" aria-label="Github" role="button" rel="noopener noreferrer external">
+                {{ partial "svg" "mini/github" }}
+            </a>
+        </p>
+        <p><a href="/privacy-policy/">Privacy Policy</a></p>
+    </div>
+</footer>
+</body>
+</html>
```

Comment:
this should not be in this partial, it's layout stuff

Reply:
In hugo you can do both way.

Converted posts and post list to use baseof